config/params/config.go

```diff
@@ -268,7 +268,8 @@ type BeaconChainConfig struct {
 	NumberOfColumns                       uint64           `yaml:"NUMBER_OF_COLUMNS" spec:"true"`                // NumberOfColumns in the extended data matrix.
 
 	// Networking Specific Parameters
-	GossipMaxSize                   uint64          `yaml:"GOSSIP_MAX_SIZE" spec:"true"`                    // GossipMaxSize is the maximum allowed size of uncompressed gossip messages.
+	DeprecatedGossipMaxSize         uint64          `yaml:"GOSSIP_MAX_SIZE" spec:"true"`                    // DeprecatedGossipMaxSize is the maximum allowed size of uncompressed gossip messages, it's deprecated and replaced by MaxPayloadSize.
```

Comment:
Marking this as deprecated. Unfortunately, I can not fully remove it, this is required because it's still used in the old testnet YAML, which is deprecated: https://github.com/eth-clients/eth2-networks  
We need to rewrite some of the config tests to use the correct config, but that shouldn't be part of this PR since it's a bigger task.